runtime/src/weights/pallet_election_provider_multi_phase.rs

```diff
@@ -0,0 +1,162 @@
+			.saturating_add(T::DbWeight::get().writes(3 as Weight))
+	}
+	// Storage: ElectionProviderMultiPhase SignedSubmissionIndices (r:1 w:1)
+	// Storage: ElectionProviderMultiPhase SignedSubmissionNextIndex (r:1 w:1)
+	// Storage: ElectionProviderMultiPhase SnapshotMetadata (r:1 w:1)
+	// Storage: ElectionProviderMultiPhase SignedSubmissionsMap (r:1 w:0)
+	// Storage: ElectionProviderMultiPhase QueuedSolution (r:1 w:1)
+	// Storage: ElectionProviderMultiPhase Round (r:1 w:1)
+	// Storage: ElectionProviderMultiPhase DesiredTargets (r:0 w:1)
+	// Storage: ElectionProviderMultiPhase Snapshot (r:0 w:1)
+	// Storage: ElectionProviderMultiPhase CurrentPhase (r:0 w:1)
+	fn elect_queued(a: u32, _d: u32, ) -> Weight {
+		(215_156_000 as Weight)
+			// Standard Error: 53_000
+			.saturating_add((1_186_000 as Weight).saturating_mul(a as Weight))
+			.saturating_add(T::DbWeight::get().reads(6 as Weight))
+			.saturating_add(T::DbWeight::get().writes(8 as Weight))
+	}
+	// Storage: ElectionProviderMultiPhase CurrentPhase (r:1 w:0)
+	// Storage: ElectionProviderMultiPhase SnapshotMetadata (r:1 w:0)
+	// Storage: TransactionPayment NextFeeMultiplier (r:1 w:0)
+	// Storage: ElectionProviderMultiPhase SignedSubmissionIndices (r:1 w:1)
+	// Storage: ElectionProviderMultiPhase SignedSubmissionNextIndex (r:1 w:1)
+	// Storage: ElectionProviderMultiPhase SignedSubmissionsMap (r:0 w:1)
+	fn submit() -> Weight {
+		(66_330_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(5 as Weight))
+			.saturating_add(T::DbWeight::get().writes(3 as Weight))
+	}
+	// Storage: ElectionProviderMultiPhase CurrentPhase (r:1 w:0)
+	// Storage: ElectionProviderMultiPhase Round (r:1 w:0)
+	// Storage: ElectionProviderMultiPhase DesiredTargets (r:1 w:0)
+	// Storage: ElectionProviderMultiPhase QueuedSolution (r:1 w:1)
+	// Storage: ElectionProviderMultiPhase SnapshotMetadata (r:1 w:0)
+	// Storage: ElectionProviderMultiPhase MinimumUntrustedScore (r:1 w:0)
+	// Storage: ElectionProviderMultiPhase Snapshot (r:1 w:0)
+	fn submit_unsigned(v: u32, t: u32, a: u32, d: u32, ) -> Weight {
+		(0 as Weight)
+			// Standard Error: 19_000
+			.saturating_add((989_000 as Weight).saturating_mul(v as Weight))
+			// Standard Error: 38_000
+			.saturating_add((651_000 as Weight).saturating_mul(t as Weight))
+			// Standard Error: 63_000
+			.saturating_add((9_673_000 as Weight).saturating_mul(a as Weight))
+			// Standard Error: 95_000
+			.saturating_add((2_907_000 as Weight).saturating_mul(d as Weight))
+			.saturating_add(T::DbWeight::get().reads(7 as Weight))
+			.saturating_add(T::DbWeight::get().writes(1 as Weight))
+	}
+	// Storage: ElectionProviderMultiPhase Round (r:1 w:0)
+	// Storage: ElectionProviderMultiPhase DesiredTargets (r:1 w:0)
+	// Storage: ElectionProviderMultiPhase MinimumUntrustedScore (r:1 w:0)
+	// Storage: ElectionProviderMultiPhase Snapshot (r:1 w:0)
+	fn feasibility_check(v: u32, t: u32, a: u32, _d: u32, ) -> Weight {
```

Comment:
It's unfortunate that the dependency on `d` was lost. Polkadot's weights look very different:
```rust
	fn feasibility_check(v: u32, t: u32, a: u32, d: u32, ) -> Weight {
		Weight::from_ref_time(0 as u64)
			// Standard Error: 4_000
			.saturating_add(Weight::from_ref_time(789_000 as u64).saturating_mul(v as u64))
			// Standard Error: 8_000
			.saturating_add(Weight::from_ref_time(41_000 as u64).saturating_mul(t as u64))
			// Standard Error: 13_000
			.saturating_add(Weight::from_ref_time(5_713_000 as u64).saturating_mul(a as u64))
			// Standard Error: 20_000
			.saturating_add(Weight::from_ref_time(1_311_000 as u64).saturating_mul(d as u64))
			.saturating_add(T::DbWeight::get().reads(4 as u64))
```